src/hooks/useQueryParams.ts

```diff
@@ -7,6 +7,9 @@ const useQueryParams = (depth?: number) => {
       ignoreQueryPrefix: true,
       depth: depth || 100,
       decoder(str) {
+        if (/^(\d+|\d*\.\d+)$/.test(str)) {
+          return parseFloat(str);
+      }
```

Comment:
this adds number support to query string. it kept importing t he following `params: [1]` as a string instead of number:

![image](https://user-images.githubusercontent.com/364566/70274553-32d04d00-1761-11ea-890c-4c372d829230.png)